routes/v3/covid-19/apiVaccine.js

```diff
@@ -105,18 +108,34 @@ router.get('/v3/covid-19/vaccine/coverage/states/:query', async (req, res) => {
 			const { timeline } = stateVaccineData;
 			const { lastdays } = req.query;
 			const numericLastDays = getLastDays(lastdays);
-			const obj = { state: stateVaccineData.state, timeline: {} };
-			Object.keys(timeline).slice(numericLastDays * -1).forEach((date) => {
-				obj.timeline[date] = timeline[date].total;
-			});
+			const obj = {
+				state: stateVaccineData.state,
+				timeline: buildVaccineTimeline(timeline, numericLastDays),
+				timelineList: buildVaccineTimelineList(timeline, numericLastDays)
+			};
 			res.send(obj);
 		} else {
 			res.status(404).send({ message: 'No vaccine data for requested state or state does not exist' });
 		}
 	} else {
 		res.status(404).send({ message: 'Error fetching vaccine state coverage data' });
 	}
+});
+
+function buildVaccineTimeline(fullTimeline, numDays) {
+	const truncatedTimeline = {};
+	Object.keys(fullTimeline).slice(numDays * -1).forEach(date => {
+		truncatedTimeline[date] = fullTimeline[date].total;
+	});
+	return truncatedTimeline;
+}
+
+function buildVaccineTimelineList(fullTimeline, numDays) {
```

Comment:
Can you add a docstring comment for this method to explain the difference between this and the above method